mcgill-en.csl

```diff
@@ -37,6 +37,18 @@
         <single>para</single>
         <multiple>paras</multiple>
       </term>
+      <term name="sub verbo" form="short">
+			  <single></single>
```

Comment:
The quick answer is "not off the top of my head." Legal sources often 
have articles which need specific reference; there might be others but I 
haven't had need of them yet in my work.

On 2015-12-14 4:31 PM, Rintze M. Zelle wrote:
>
> In mcgill-en.csl 
> <https://github.com/citation-style-language/styles/pull/1826#discussion_r47510654>:
>
> > @@ -37,6 +37,18 @@
> >          <single>para</single>
> >          <multiple>paras</multiple>
> >        </term>
> > +      <term name="sub verbo" form="short">
> > +			  <single></single>
>
> @liammr <https://github.com/liammr>, by the way, are you missing any 
> other locator terms besides "article"? Frank Bennett already added 
> "article", "rule", and "title" to his fork of CSL for improved legal 
> support, and I was wondering if you need any others.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/citation-style-language/styles/pull/1826/files#r47510654>.
>

-- 
Liam McHugh-Russell, JD, LLM
Researcher, Department of Law
European University Institute, Villa Schifanoia
Via Boccaccio, 121
I - 50133 Firenze, Italy
+39 33 4811 4340